File: WanaD.AppServiceContract/PlotScoreData.cs
using Fs.Erp.Core;
using System;
using System.Collections.Generic;

namespace WanaD.AppServiceContract
{
    public class PlotScoreData
    {
        public Guid? Id { get; set; }
        public FarmAndPlotData Farm { get; set; }
        public NameAndIdData Plot { get; set; }
        public decimal? TotalScore { get; set; }
        public DateTime? Date { get; set; }
        public string Result { get; set; }
        public string VerifierName { get; set; }
        public DateTime? VerifierDate { get; set; }
        public string StatusCode { get; set; }
        public string StatusName { get; set; }
        public string MonsoonStaff { get; set; }
        public string Remark { get; set; }
        public List<FileInfoData> FileInfoList { get; set; }
        public Guid? WorkflowDocumentId { get; set; }
    }
}

File: WanaD.WebClient/app/src/Modules/WanaD/Scripts/AppServiceContract/PlotScoreData.ts

 

 

 

    import FarmAndPlotData from 'Modules/WanaD/Scripts/AppServiceContract/FarmAndPlotData';

    import NameAndIdData from 'Modules/WanaD/Scripts/AppServiceContract/NameAndIdData';

 
class PlotScoreData 
{
    
        Id: string;
    
        Farm: FarmAndPlotData;
    
        Plot: NameAndIdData;
    
        TotalScore: number;
    
        Date: Date;
    
        Result: string;
    
        VerifierName: string;
    
        VerifierDate: Date;
    
        StatusCode: string;
    
        StatusName: string;
    
        MonsoonStaff: string;
    
        Remark: string;
    
        FileInfoList: any;
    
        WorkflowDocumentId: string;
    
}
 
export default PlotScoreData;
 




